dev/Interop/CameraCaptureUI/CameraCaptureUI/CameraCaptureUI.cpp

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#include "pch.h"
+#include <future>
+#include <ppltasks.h>
+#include <pplawait.h>
+#include <ShlObj.h>
+#include "CameraCaptureUI.h"
+#include "Microsoft.Windows.Media.Capture.CameraCaptureUI.g.cpp"
+#include "CameraCaptureUITelemetry.h"
+#include "TelemetryHelper.h"
+
+//using namespace winrt::Windows::Media::Capture;
+using namespace winrt::Windows::UI;
+using namespace winrt::Windows::Storage;
+using namespace winrt::Windows::System;
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::Foundation::Collections;
+using namespace winrt::Windows::ApplicationModel::DataTransfer;
+
+namespace winrt::Microsoft::Windows::Media::Capture::implementation
+{
+    IAsyncOperation<hstring> LaunchCameraForResultToken(winrt::Microsoft::UI::WindowId const& targetWindow, ValueSet const& properties)
+    {
+        LauncherOptions options;
+        if (targetWindow.Value)
+        {
+            check_hresult(options.as<IInitializeWithWindow>()->Initialize(reinterpret_cast<HWND>(targetWindow.Value)));
+        }
+
+        options.TreatAsUntrusted(false);
+        options.DisplayApplicationPicker(false);
+        options.TargetApplicationPackageFamilyName(L"Microsoft.WindowsCamera_8wekyb3d8bbwe");
```

Comment:
Consider having this application package name and other strings stored in static consts.

Reply:
Makes sense, updated!